Edits to Git entries (11 total) (#562)
Codecademy/docs: content/git/concepts/pull/pull.md

In Git, pulling is the process of moving code from one repository to another one. Often this is used to move code from a remote machine to a local one, or retrieving code from a repository hosting service like GitHub.

Pulling a repository allows a programmer to access code from a remote backup and make it available on their local machine. It is also used when collaborating with others in order to keep the local code up to date with the team's current version.

## Syntax

> The basic syntax for pulling in Git is this:

Reviewer: ```suggestion
The syntax for pulling changes in Git is this:
```

  nothing to commit, working tree clean
  ```

- Having a valid remote repository set. In this case the remote repository is named `origin` and can be found at the url `https://github.com/CompanyName/product-dev`:

  ```shell
  $ git remote -v

  origin    https://github.com/CompanyName/product-dev.git (fetch)
  origin    https://github.com/CompanyName/product-dev.git (push)
  ```

> At this stage, pulling is as simple as using the basic syntax with the correct remote and branch names:

Reviewer: ```suggestion
At this stage, pulling can accomplished with the names of a valid remote and branch:
```

Reviewer 2: Edits:
- can be accomplished*
- "the names of a valid remote and branch:" -> "a valid remote and branch name:"
